Signal/src/UserInterface/Strings.swift

```diff
@@ -1,12 +1,14 @@
-//  Created by Michael Kirk on 12/29/16.
-//  Copyright © 2016 Open Whisper Systems. All rights reserved.
+//
+//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//
 
 import Foundation
 
-@objc class Strings: NSObject {
-    @objc class Calls: NSObject {
-        static let callBackButtonTitle = NSLocalizedString("CALLBACK_BUTTON_TITLE", comment: "notification action")
-        static let missedCallNotificationBody = NSLocalizedString("MISSED_CALL", comment: "notification title")
-        static let missedCallNotificationBodyWithCallerName = NSLocalizedString("MSGVIEW_MISSED_CALL", comment: "notification title. Embeds {{Caller's Name}}")
-    }
+/**
+ * Strings re-used in multiple places should be added here.
+ */
+@objc class CallStrings: NSObject {
+    static let callBackButtonTitle = NSLocalizedString("CALLBACK_BUTTON_TITLE", comment: "notification action")
+    static let missedCallNotificationBody = NSLocalizedString("MISSED_CALL", comment: "notification title")
+    static let missedCallNotificationBodyWithCallerName = NSLocalizedString("MSGVIEW_MISSED_CALL", comment: "notification title. Embeds {{Caller's Name}}")
```

Comment:
I thought it over and remembered that it can be a better practice (imo) to not try to reuse strings _unless_ they're something really basic like "OK", "CANCEL", etc. Other strings - though similar in one language, might be need different translations in different languages. Sometimes that's because a language like English will use the same word for different parts of speech ("count" is a verb and a noun, for example). Another example is that more verbose languages like Italian often have longer copy and you'll need to abbreviate or reword it in some contexts (a cramped button, for example) but not others.